api/turing/cmd/main.go

```diff
@@ -66,6 +67,12 @@ func main() {
 	db.LogMode(false)
 	defer db.Close()
 
+	// Initialise NewRelic
```

Comment:
Nitpick: Can this come after the Sentry initialisation so that the somewhat similar steps are placed together?

Reply:
Unfortunately I can't because it needs to intialise the NewRelic before the handler initialisation is done. If not this becomes the implementation of newrelic (https://github.com/gojek/mlp/blob/main/api/pkg/instrumentation/newrelic/newrelic.go#L12)